charts/gardener/gardenlet/templates/deployment.yaml

```diff
@@ -62,6 +62,7 @@ spec:
 {{ include "gardenlet.deployment.labels" . | indent 8 }}
         projected-token-mount.resources.gardener.cloud/skip: "true"
         seccompprofile.resources.gardener.cloud/skip: "true"
+        pod-topology-spread-constraints.resources.gardener.cloud/skip: "true"
```

Comment:
The label has another key: https://github.com/gardener/gardener/blob/73c9272bcf13e5ab610955c38334321a589c2213/pkg/apis/resources/v1alpha1/types.go#L137